src/desktop_notifier/macos_support.py

```diff
@@ -71,4 +75,12 @@ def is_signed_bundle() -> bool:
         None,
     )
 
-    return cast(int, signed_status) == 0
+    if cast(int, signed_status) == 0:
+        return True
+    else:
+        _log_unsigned_warning(f"signed_status is {signed_status}")
+        return False
+
+
+def _log_unsigned_warning(msg: str) -> None:
```

Comment:
I don't think it's worth extracting this common prefix into a helper function, mainly because "Unsigned bundle" is misleading at least for the first usage of this function.

Reply:
i adjusted the name of the fxn slightly and got rid of the other call; let me know if that works for you

Reply:
Works for me, thanks!